lib/deck.rb

```diff
@@ -0,0 +1,45 @@
+class Deck 
+  attr_reader :high_ranking_card, :percent_high_ranking
+  attr_accessor :cards, :rank_of_card_at
+
+  def initialize(cards)
+    @cards = cards
+  end 
+
+  def rank_of_card_at(index)
+    @cards[index].rank
+  end 
+
+  def high_ranking_card
+    high_cards = []
+    @cards.each do |card| 
+      if card.rank >= 11
+        high_cards << card
+      end 
+    end
+    high_cards
```

Comment:
This works! Is there another enumerable you could use instead of `each`, so that you wouldn't need the `high_cards` accumulator variable?